Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1735,6 +1735,12 @@ def send_mail(emailto, emailfrom, subject, body, entry_ids, cc, bcc, htmlBody, r
     message['subject'] = header(subject)
     message['reply-to'] = header(replyTo)
 
+    # The following headers are being used for the reply-mail command.
+    if inReplyTo:
+        message['In-Reply-To'] = header(' '.join(inReplyTo))
+    if references:
```

Comment:
what does `references` do?

Reply:
Similar to `In-Reply-To`, you refer to the message you're replying to but this header can also contain more message ids in order to create a "threaded conversation" and see all the messages that are linked to this one so one could find it on his mailbox.
https://tools.ietf.org/html/rfc2822#appendix-A.2